projects/1.16/assets/controlling/controlling/lang/zh_cn.json

```
@@ -5,11 +5,11 @@
   "options.showNone": "显示未绑定按键",
   "options.availableKeys": "可用按键",
   "options.sort": "排序",
-  "options.category": "分类",
+  "options.category": "类别",
   "options.key": "按键",
-  "options.sortNone": "空按键",
+  "options.sortNone": "无",

```

```suggestion
  "options.sortNone": "默认",
```
原文意思是「无」，但是放在排序中就是使用「默认」排序，不确定使用哪种来替代「空按键」

---

None可能是想表述“不作改动”
从表意层面上来看“默认”应该比较好

（为什么发了三个......）